tests/test_histeq.py

```diff
@@ -1 +1,22 @@
-# Last names: P
\ No newline at end of file
+# Last names: P
+
+from minecraft_gen import histeq
+import numpy as np
+from skimage import exposure
+
+def test_histeq():
+  img_test = np.ones((3,3))
+  alpha = 1
+  
+  hist_test = histeq(img_test, alpha)
+  
+  #copied code from histeq to ensure output is consistent with previous version
+  img_cdf, bin_centers = exposure.cumulative_distribution(img_test)
+  img_eq = np.interp(img_test, bin_centers, img_cdf)
+  img_eq = np.interp(img_eq, (0, 1), (-1, 1))
+  hist_compare = alpha * img_eq + (1 - alpha) * img_test
```

Comment:
I'd probably just make a copy of the output and compare that, but this is also a regression test. The downside to this is that if scikit-image changed "exposure" or numpy changed "interp" (unlikely!), this regression test wouldn't catch that. It would only protect you if you refactor code (which is important).

Usually you'd use this sort of thing with property-based testing, where you input random values and verify the result behaves like the computation above.